code/modules/clothing/masks/gasmask.dm

```diff
@@ -149,6 +150,8 @@
 	icon_state = "plaguedoctor"
 	inhand_icon_state = "gas_mask"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 2,ENERGY = 2, BOMB = 0, BIO = 75, FIRE = 0, ACID = 0)
+	flags_cover = MASKCOVERSEYES
+	tint = 0
 
 /obj/item/clothing/mask/gas/syndicate
```

Comment:
Shouldn't syndie masks have a reduced tint value, being (mostly) exclusive to nukies and tacticool?

Reply:
you're right, i'll lower it to 0.25 so that effect doesn't apply